project/Build.scala

```diff
@@ -190,19 +191,6 @@ object Scrooge extends Build {
       finagle("core") exclude("org.mockito", "mockito-all"),
       finagle("thrift") % "test"
     ),
-    libraryDependencies <++= scalaVersion({
```

Comment:
So the new version works because mustache.java 0.8.17 works for scala 2.11 (except the extensions), right?

For future reference, what's not working with the old approach?